api-server/web-server/src/main.rs

```diff
@@ -37,8 +40,13 @@ async fn main() {
     let args = ApiServerWebServerConfig::parse();
     log::info!("Command line options: {args:?}");
 
+    // TODO: generalise network configuration
```

Comment:
"generalize". We use American English in the code.

Reply:
Muscle memory... I'm sure there's an Emacs macro to fix my spelling 😄